backend/src/config.py

```diff
@@ -1,27 +1,41 @@
 from enum import Enum
 from pathlib import Path
+
+from src.data_provider.FloydExtractorConfiguration import FloydExtractorConfiguration
 from src.solver.ConnectionSolverConfiguration import ConnectionSolverConfiguration
 
 
-FLOYD_EXTRACTOR_PERIOD_MULTIPLIER: float = 0.5
-FLOYD_EXTRACTOR_CHANGE_PENALTY = 120  # sec
 MAX_WALKING_TIME_IN_MINUTES: float = 10
 MAX_WALKING_DISTANCE = 1000  # min
 WALKING_SPEED: float = 1.0
 WALKING_ROUTE_ID: int = -1
 DATETIME_FORMAT: str = "%Y-%m-%d %H:%M:%S"
+CHANGE_PENALTY = 120  # sec
+
+DEFAULT_FLOYD_EXTRACTOR_CONFIGURATION = FloydExtractorConfiguration(
+    daily_period_multiplier=0.5,
+    nightly_period_multiplier=0.25,
+    change_penalty=CHANGE_PENALTY,
+    nightly_route_ranges=[(600, 699), (900, 999)],
```

Comment:
we could use a predicate (as function or lambda), e.g.:
```py
is_night_route_func=lambda name: len(name) >= 2 and (name.startswith('6') or name.startswith('9'))
```
but your solution has a better performance